File: 01_Basico/02 - Controle_Selecao/Respsotas_em_Python/questao09.py
pergunta1 = input("Você telefonou para a vitima ? [S]im ou [N]ão")
pergunta2 = input("Você este no local do crime? [S]im ou [N]ão")
pergunta3 = input("Você mora perto da vítima ? [S]im ou [N]ão")
pergunta4 = input("Você devia para a vítima? [S]im ou [N]ão")
pergunta5 = input("Você já trabalhou com a vítima ?  [S]im ou [N]ão")
nivel_suspeito = 0

if pergunta1[0].upper() == "S":
    nivel_suspeito += 1
if pergunta2[0].upper() == "S":
    nivel_suspeito += 1
if pergunta3[0].upper() == "S":
    nivel_suspeito += 1
if pergunta4[0].upper() == "S":
    nivel_suspeito += 1
if pergunta5[0].upper() == "S":
    nivel_suspeito += 1
    
if nivel_suspeito == 2:
    print("Suspeito")
elif nivel_suspeito == 3 or nivel_suspeito == 4:
    print("Cúmplice")
elif nivel_suspeito == 5:
    print("Assassino")
else:
    print("Inocente")

File: 01_Basico/02 - Controle_Selecao/Resposta_em_JavaScript/questao09.js
var pergunta1 = prompt("Você telefonou para a vítima? [S]im ou [N]ão: ");
var pergunta2 = prompt("Você esteve no local do crime? [S]im ou [N]ão: ");
var pergunta3 = prompt("Você mora perto da vítima? [S]im ou [N]ão: ");
var pergunta4 = prompt("Você devia para a vítima? [S]im ou [N]ão: ");
var pergunta5 = prompt("Você já trabalhou com a vítima? [S]im ou [N]ão: ");

let nivel_suspeito = 0;

if (pergunta1[0].toUpperCase() == "S"){
    nivel_suspeito += 1;
}
if (pergunta2[0].toUpperCase() == "S"){
    nivel_suspeito += 1;
}
if (pergunta3[0].toUpperCase() == "S"){
    nivel_suspeito += 1;
}
if (pergunta4[0].toUpperCase() == "S"){
    nivel_suspeito += 1;
}
if (pergunta5[0].toUpperCase() == "S"){
    nivel_suspeito += 1;
}

switch(nivel_suspeito){
    case 2:
        console.log("Suspeito");
        break;
    case 3:
    case 4:
        console.log("Cúmplice");
        break;
    case 5:
        console.log("Assassino");
        break;
    default:
        console.log("Inocente");
}
